postmaster/apiv1/admins.py

```diff
@@ -5,8 +5,10 @@
 an admin to create, delete, and update admins
 """
 
-from flask import request
+from flask import request, jsonify
```

Comment:
jsonify should not be used because we have the json_wrap decorator which will jsonify dictionaries and SQLAlchemy queries